### DISCUSSION TITLE
11월 셋째 주 스프린트 [11/15 - 11/21]

### DISCUSSION THREAD

**Discussion by ghost** [team-mokcha/corabal-beta#44] [스프린트 일지]
# 성과
11/15
- 네비게이션 컴포넌트 아톰 단위까지 나누기

11/16
- 외부링크 연결 버튼 재사용 가능하게 리팩토링

# 할 일
- [x] 네비게이션 컴포넌트 아톰 단위까지 나누기
- [x] 외부링크 연결 버튼 재사용 가능하게 리팩토링
- [x] 레코드 스크린 리팩토링
- [ ] 버튼은 시간나면 리팩토링하기

# 기타
- 11/17 수 휴식 (수연 면접)
- 11/18 목 미정 (수연 일정 있으나 수요일에 봐야 할 듯)
- 11/20 토 휴식 (수연 일정)

**Comment by HyejinYang** [comment]
### 논의 필요
- 버튼 디자인 : 기기 너비에 따라서 record button의 margin 값을 피그마와 동일하게 적용해도 원디자인의 가로 세로 비율이 맞춰지지 않는 이슈 존재
   -> 디자이너와 논의해서 결정해야 함
@alveloper

**Comment by ghost** [comment]
11/19
- 레코드 스크린 + 백엔드 로직 때문에 혜진님이 수요일까지 실험해보시겠다고 하심

**Reply by ghost** [reply]
수연: 화요일 오후에 시간 ok, 안 되면 금요일도 ok

**Reply by HyejinYang** [reply]
개인 일정 문제로 금요일까지로 변경해 진행함. 해당 리팩 관련 자세한 사항은 이슈 https://github.com/team-mokcha/corabal-beta/issues/45 참고.

**Comment by ghost** [comment]
다다음주에 위키 정리
